gregorypoole.txt

```diff
@@ -0,0 +1,127 @@
+https://codepen.io/gregory-poole/pen/OJPwpdV?editors=1100
+
+
+1. What is Semantic HTML? 
+A way to organize html to make it make sense. 
+
+    2. What is HTML used for? 
+        html is used to create web pages
+
+    3. What is an attribute and where do we put it? 
+        a attribute can put a link inside of a tag.
+
+    4. What is the h1 tag used for? How many times should I use it on a page?
+    A h1 tag is used for a heading 
+
+    5. Name two tags that have required attributes
+    href and img src 
+
+    6. What do we put in the head of our HTML document? 
+    heading
+
+    7. What is an id? 
+    an id creates a selector for CSS
+
+    8. What elements can I add an id to? 
+        any tags 
+
+    9. How many times can I use the same id on a page? 
+     you can only use a id once 
+
+    10. What is a class? 
+    a class is a descriptor for more than one tag
+
+    11. What elements can I add a class to? 
+    p, div, h1, h2, etc..
+
+    12. How many times can I use the same class on a page? 
+    as many as you want 
+    \
+    13. How do I get my link to open in a new tab?
+    add a blank to your a tag 
+
```

Comment:
`blank` is part of it, can you be more specific? (the actual attribute and value that you add to your `<a>` tag)